tests/phpunit/WC_Stripe_Helper_Test.php

```
@@ -1349,4 +1349,86 @@ public function provide_test_build_line_items(): array {
+		} else {
+			unset( $new_settings['express_checkout_button_locations'] );
+		}
+
+		WC_Stripe_Helper::update_main_stripe_settings( $new_settings );
+
+		$result = WC_Stripe_Helper::should_load_scripts_for_ece_location( $location );
+
+		WC_Stripe_Helper::update_main_stripe_settings( $original_settings );
+
+		$this->assertSame( $expected, $result );
+	}
```

PHPMD (2.15.0)

[error] The method test_should_load_scripts_for_ece_location is not named in camelCase. (undefined)

(CamelCaseMethodName)